### PR TITLE
Openvpn v2.5.2-r0

### LINKED ISSUE #2: Openvpn multi-arch image addition
In KKP, OpenVPN is deployed in the user cluster, but its image doesn't support multi-arch and is not kept up to date. That image (contained in `/hack/image/openvpn`) should be moved to this repository. The image here maintained is supposed to be multi-arch, and kept up to date.

This issue is part of the epic https://github.com/kubermatic/kubermatic/issues/6584

### PR DESCRIPTION
**Pull request by mlavacca**
**What this PR does / why we need it**:
Repository initialization to build multi-arch images of OpenVPN. The first version is the `2.5.2-r0`, the one present in the alpine 3.13 repository.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #2 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: hack/ci/create-pull-request.sh [hunk 1]
```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env bash
```
Comment (xrstf) [inline hack/ci/create-pull-request.sh]: I don't think this kind of script makes sense for this repo. I assume you copied it from the kubelet repo?

Comment (mlavacca) [reply hack/ci/create-pull-request.sh]: yep, deleted it

**Comment by mlavacca** [comment]
PTAL @xrstf

**Review by kron4eg — APPROVED** [review]

**Pull request merged**